package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "scripts": {
+    "lint": "gulp lint",
+    "lint:fix": "gulp lint --fix",
+    "test": "npm run gulp lint"
+  },
+  "devDependencies": {
+    "@babel/eslint-parser": "^7.15.4",
+    "@league-of-foundry-developers/foundry-vtt-types": "^9.280.0",
```

Comment:
this is purely included for convenience--most IDEs will pick up the types from the installed dependency, and provide autocomplete/etc. for the project